geometry/geometry_config.py

```diff
@@ -389,46 +389,10 @@
               c.muShield.dZ5 + c.muShield.dZ6 +
               c.muShield.dZ7 + c.muShield.dZ8
         ) + c.muShield.LE
-        c.muShield.z = -(c.decayVolume.length + c.muShield.length) / 2.
-
-    if muShieldDesign == 3:
-     c.muShield.dZ1 = 3.5*u.m
-     c.muShield.dZ2 = 5.*u.m
-     c.muShield.dZ3 = 3.5*u.m
-     c.muShield.dZ4 = 2.0*u.m
-     c.muShield.dZ5 = 1.*u.m
-     c.muShield.dZ6 = 3.*u.m
-     c.muShield.dZ7 = 3.*u.m
-     c.muShield.dZ8 = 3.*u.m
-     c.muShield.dXgap = 0.2*u.m
-
-    if muShieldDesign == 2:
-     c.muShield.dZ0 = 0*u.m      #  extra hadron absorber
-     c.muShield.dZ1 = 2.5*u.m
-     c.muShield.dZ2 = 3.5*u.m
-     c.muShield.dZ3 = 3.0*u.m
-     c.muShield.dZ4 = 3.0*u.m
-     c.muShield.dZ5 = 2.5*u.m
-     c.muShield.dZ6 = 2.5*u.m
-     c.muShield.length = 2*(c.muShield.dZ1+c.muShield.dZ2+c.muShield.dZ3+c.muShield.dZ4+
-                         c.muShield.dZ5+c.muShield.dZ6) + c.muShield.LE  # leave some space for nu-tau detector   
-    # for passive design, fDesign==1
-    if muShieldDesign == 1:
-        c.muShield.length =  70*u.m 
-        c.muShield.z  =  -c.decayVolume.length/2.-c.muShield.length/2. - c.muShield.LE  # leave some space for nu-tau 
-    if muShieldDesign == 3 or muShieldDesign == 4 or muShieldDesign == 5: 
-     c.muShield.length = 2*(c.muShield.dZ0+c.muShield.dZ1+c.muShield.dZ2+c.muShield.dZ3+c.muShield.dZ4+c.muShield.dZ5+c.muShield.dZ6
-                         +c.muShield.dZ7+c.muShield.dZ8 ) + c.muShield.LE  # leave some space for nu-tau 
-     c.muShield.z  =  -c.decayVolume.length/2.-c.muShield.length/2.
-    if muShieldDesign == 6: 
-     c.muShield.length = 2*(c.muShield.dZ1+c.muShield.dZ2+c.muShield.dZ3+c.muShield.dZ4+c.muShield.dZ5+c.muShield.dZ6
-                         +c.muShield.dZ7+c.muShield.dZ8 ) + c.muShield.LE  # leave some space for nu-tau 
-     c.muShield.z  =  -c.decayVolume.length/2.-c.muShield.length/2.
+        c.muShield.z = -(c.decayVolume.length + c.muShield.length) / 2. + 1.6*u.m
```

Comment:
@yabezsh  This line seems to be the culprit. I don't understand why this only seems to be a problem for the combi-shield (or muon shield 8, but if I disable it, the target is in the right spot.

Can't run an overlap check until this evening unfortunately.

Reply:
@yabezsh Reverting the change to this line also fixes the overlaps with between target and absorber.

Reply:
Interesting... it seems that if we don't use option 8 of muShield, this line just moves Shield and everything behind it closer to the SND area. However, with option 8 it doesn't move closer to the SND area at all, instead the target is pushed a bit inside Shield. It seems that the position variables for the two shield options are related in different ways

Reply:
Yes, and I really can't figure out why. I feel like I'm missing something really obvious somewhere...